api/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = seismic_site.settings
+DJANGO_CONFIGURATION=Test
```

Comment:
pytest setup is already done in `setup.cfg` - https://github.com/code4romania/seismic-risc/blob/master/api/setup.cfg#L26

Reply:
I was getting this error, that's why I had to add the `pytest.ini`

```
django.core.exceptions.ImproperlyConfigured: Requested setting INSTALLED_APPS, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
```

Am I running it wrong? `docker exec seismic-risc-api pytest`

Reply:
That is happening because you need to run the tests through `docker-compose`, otherwise the container won't have the environment variables available. 

You can either run the tests using `make test` or `docker-compose run --rm api "pytest"`. 

You can use `exec` only if your container is up and it would look something like `docker-compose exec api pytest`

Reply:
Got it, thanks.

---

The PR got merged meanwhile.
The `ini` file is redundant but does give a bit of convenience in allowing tests to run outside of compose. I'd say let's keep it for now.